packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -28,16 +29,19 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
     );
     const styles = useStyles2(getCheckboxStyles);
 
+    const ariaChecked = indeterminate ? 'mixed' : undefined;
```

Comment:
this could be a good stage to add the aria-checked for checked/unchecked as well. what do you think?

Reply:
I don't _think_ it's nessicary because UAs should get that state from the checked property of the checkbox